### PR TITLE
Review of stakeholders

### PR DESCRIPTION
**Pull request by Silvan-WMDE**
- Merge Large Non Wikimedia Wikibase Project and Data Importer
 - Add Wikipedia reader in Client
 - Add Tool Builder in Overview
 - Replace duplicate Researcher in Overview with link to Repo Researcher

Bug: [T273973](https://phabricator.wikimedia.org/T273973), [T274247](https://phabricator.wikimedia.org/T274247)

### CODE REVIEW

File: Introduction_and_Goals.md [hunk 1]
```diff
@@ -36,16 +36,16 @@ See [WikibaseClient editor](./systems/WikibaseClient/01-Introduction_and_Goals.m
...
-- Quickly create visualizations based on data from Wikidata without the need to download it.
+See [Researcher](./systems/WikibaseRepo/01-Introduction_and_Goals.md#researcher)
 
 ### Non Wikimedia Wikibase Project
 
 - Create knowledge bases similar to Wikidata using our same software stack (including extensions, tools and features)
 - Install, configure, update, and maintain a set of Wikibase systems with confidence.
 
-#### Large Non Wikimedia Wikibase Project
+### Third-party Tool Builder
```
Comment (tzhelyazkova) [inline Introduction_and_Goals.md:46]: Why "Third-party"? Do you mean to imply it's volunteers who build the tools?

File: systems/WikibaseClient/01-Introduction_and_Goals.md [hunk 2]
```diff
@@ -32,3 +32,9 @@ Wikibase Client is part of [Wikibase](../../Introduction_and_Goals.md). It is a
 - Have data updates which cause pages changes to be shown in [Watchlists](../../Glossary.md#watchlist), when the page being changed is watched.
 - Update Wikidata’s data used on Wikipedia when it is incorrect or outdated via web interface.
 - Find wrong or missing data on a Wikipedia article and correct or add it to Wikidata.
+
+### Wikipedia reader
+
+- Have a structured representation of facts about the world in Wikipedia articles (e.g. infoboxes)
+- Rely on centrally maintained data and trust that it is up-to-date, even on small Wikipedias
+- Find reliable and up-to-date references for facts about the world
```
Comment (lydiapintscher) [inline systems/WikibaseClient/01-Introduction_and_Goals.md:40]: We generally don't talk about facts in Wikidata and Wikibase as that implies a notion of truth. We don't. We focus on verifiability, so "claims" or "data".

File: systems/WikibaseRepo/01-Introduction_and_Goals.md [hunk 3]
```diff
@@ -29,6 +29,10 @@ Wikibase Repository is part of [Wikibase](../../Introduction_and_Goals.md). Wiki
 
 - Edit [Terms](../../Glossary.md#term) and sitelinks of entities without using JavaScript in a browser.
 
+### Wikidata reader
+
+- Look up and explore facts about the world
```
Comment (lydiapintscher) [inline systems/WikibaseRepo/01-Introduction_and_Goals.md:34]: Similar to above. We avoid "facts".

File: Introduction_and_Goals.md (hunk 1 above)
Comment (Silvan-WMDE) [reply Introduction_and_Goals.md]: Ok, _Tool Builder_ it is

File: systems/WikibaseRepo/01-Introduction_and_Goals.md (hunk 3 above)
Comment (Silvan-WMDE) [reply systems/WikibaseRepo/01-Introduction_and_Goals.md]: After reading [your comment on the phab ticket](https://phabricator.wikimedia.org/T273973#6826103): Should I remove this completely?

Comment (lydiapintscher) [reply systems/WikibaseRepo/01-Introduction_and_Goals.md]: I think I'd remove it, yeah.

**Review by tzhelyazkova — APPROVED** [review]

**Pull request merged**